### ISSUE TITLE
OrderingType is not converted to string during query execution

### ISSUE THREAD

**Issue by Bktero** [ravendb/RavenDB-Python-Client#120]
Hello

Here is a minimalist code that reproduces the issue:

```python
from pyravendb.store import document_store
from pyravendb.data.query import OrderingType

store = document_store.DocumentStore(urls=['http://127.0.0.1:8080'], database='my_database')
store.initialize()

with store.open_session() as session:
    query = session.query(collection_name='flowrates').order_by_descending('flow_rate', ordering=" AS long").take(1)
    results = list(query)
    print(results)

with store.open_session() as session:
    query = session.query(collection_name='flowrates').order_by_descending('flow_rate', ordering=OrderingType.long).take(1)
    results = list(query)
    print(results)
```

The query is successful while the second raises an exception:

```python
Traceback (most recent call last):
  File "D:\gitlab\minimalist.py", line 14, in <module>
    results = list(query)
  File "C:\Users\me\AppData\Local\Programs\Python\Python39\lib\site-packages\pyravendb\store\session_query.py", line 101, in __iter__
    return iter(self._execute_query())
  File "C:\Users\me\AppData\Local\Programs\Python\Python39\lib\site-packages\pyravendb\store\session_query.py", line 781, in _execute_query
    query = self._build_query()
  File "C:\Users\me\AppData\Local\Programs\Python\Python39\lib\site-packages\pyravendb\store\session_query.py", line 137, in _build_query
    return "".join(query_builder)
TypeError: sequence item 4: expected str instance, OrderingType found
```

Here is a patch that solves the issue:

![image](https://user-images.githubusercontent.com/1047370/154913610-f33a498b-fe43-494d-ab57-d4b8af77aeb0.png)

Is this really a bug or am I using the API incorrectly?

More details about my environment:

```bat
> python --version
Python 3.9.2
> pip list | grep raven
pyravendb                    5.0.0.2
```

I am running Windows 10.

**Comment by poissoncorp** [comment]
Thanks for creating this issue @Bktero, you are using the API correctly.
We've fixed this bug and released a [new version of the 5.0 client (5.0.0.4)]( https://pypi.org/project/pyravendb/5.0.0.4/) available on PyPI :>

**Comment by Bktero** [comment]
This solves the issue. Great! :)
